src/Persistence/MassTransit.EntityFrameworkCoreIntegration.Tests/Migrations/Audit/AuditDbContextModelSnapshot.cs

```diff
@@ -12,14 +13,17 @@ partial class AuditDbContextModelSnapshot : ModelSnapshot
     {
         protected override void BuildModel(ModelBuilder modelBuilder)
         {
+#pragma warning disable 612, 618
             modelBuilder
-                .HasAnnotation("ProductVersion", "1.1.1")
+                .HasAnnotation("ProductVersion", "2.2.6-servicing-10079")
```

Comment:
What is this?

Reply:
I think the EF he used has a higher version, it was replaced by EF automatically

Reply:
So, should this be 6.0.0 to match up with MassTransit's version? Or just increment it to 1.2.0 or something.

Reply:
I'm not sure, this value is set by EF tools, so you haven't any control on it

Reply:
I believe any migrations added after the commit highlighted on the image below would update the product version.

![image](https://user-images.githubusercontent.com/11743735/69253413-9dd72c80-0b82-11ea-90ee-b2f002ac2b37.png)

Below is the init migration add from 7/10/2017.

![image](https://user-images.githubusercontent.com/11743735/69253742-1dfd9200-0b83-11ea-8252-5bd415ad9c3d.png)


Would you like any action here?